docs/_layouts/tutorial.html

```diff
@@ -10,13 +10,25 @@
   {% include tag_manager.html %}
   <div class="wrapper">
     <div class="panel">
-      {% include en/sidebar.html %}
+      {% if page.lang == "ja-jp" %}
+        {% include jp/sidebar.html %}
+      {% else %}
+        {% include sidebar.html %}
+      {% endif %}
     </div>
 
 
     <div class="content">
         <div class="mainheader">
-          {% include en/header.html %}
+          {% if page.lang == "ja-jp" %}
+            {% include jp/header.html %}
+          {% else %}
+            {% include header.html %}
+          {% endif %}
+        </div>
+
+        <div class="tutorial-nav">
+          <ul class="tutorial-nav-list"></ul>
```

Comment:
The script just pulls the `<h3>` tags and linkifies them to insert them into the list. This was an attempt to make it so authors of tutorials wouldn't have to build a table of contents with permalinks in the front matter of every tutorial since that seems like a layout issue rather than a content one.

I agree that this logic would __ideally__ be in the template. Do you have any ideas to move the generation logic into`tutorial.html`?